models/public/hbonet-0.25/model.yml

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  "hbonet-0.25" is one of HBONet architectures <https://arxiv.org/abs/1908.03888>
+task_type: classification
+files:
+  - name: hbonet_0_25.py
+    size: 12548
+    sha256: f272c67c5910cabca576a6001089b050ae995f6e340d9b3033a2e787c5fbc1ad
+    source: https://raw.githubusercontent.com/d-li14/HBONet/master/models/imagenet/hbonet.py
+  - name: hbonet_0_25.pth
+    sha256: a69228ef353765879e929ada789a5c74dfd14fedaff931103ee1e285019dd3a0
+    size: 7863957
+    source: https://raw.githubusercontent.com/d-li14/HBONet/master/pretrained/hbonet_0_25.pth
+framework: pytorch
+postprocessing:
+  - $type: regex_replace
+    file: hbonet_0_25.py
+    pattern: 'width_mult=1.'
+    replacement: 'width_mult=0.25'
+conversion_to_onnx_args:
+  - --model-name=hbonet
+  - --model-path=$dl_dir
+  - --weights=$dl_dir/hbonet_0_25.pth
+  - --import-module=hbonet_0_25
+  - --input-shape=1,3,224,224
```

Comment:
This shape differs from documentation. Please, recheck which layout, NHWC or NCHW, is used in original model

Reply:
Replaced